bin/taste-tester

```diff
@@ -418,6 +424,20 @@ MODES:
 
   parser.parse!
 
+  if ARGV
+    if [:test, :untest, :runchef, :keeptesting].include?(mode.to_sym)
+      if options[:split]
+        options[:servers] = ARGV.map { |s| s.split(/,/) }.flatten
+      else
+        options[:servers] = ARGV
+      end
+    end
+  else
+    puts "Mode #{mode} does not accept extra arguments: #{ARGV}\n\n"
+    puts parser
+    exit(1)
```

Comment:
you exist if there's NOT an ARGV? seems wrong.

Reply:
Oh right that else goes one level up, sorry. Nice catch!